specification/dataprotection/resource-manager/readme.md

```diff
@@ -46,6 +46,14 @@ semantic-validator: true
 message-format: json
 ```
 
+### Tag: package-2022-10-preview
```

Comment:
the SDK generation failure may be caused by the inconsistence of default tag?  it should be package-2022-10-preview instead of package-preview-2022-10?

Reply:
@xiaoxuqi-ms Sure, I have changed the default tag to package-2022-10-preview